bitnami/redmine/templates/deployment.yaml

```diff
@@ -96,17 +96,15 @@ spec:
           args: {{- include "common.tplvalues.render" (dict "value" .Values.args "context" $) | nindent 12 }}
           {{- end }}
           env:
-            {{- if eq .Values.databaseType "mariadb" }}
-            - name: REDMINE_DB_MYSQL
-            {{- else }}
-            - name: REDMINE_DB_POSTGRES
-            {{- end }}
+            - name: REDMINE_DATABASE_TYPE
+              value: {{ default "" .Values.databaseType | quote }}
```

Comment:
Why do you set a default with an empty string? The default value should come from the **values.yaml**

Reply:
Fixed